runtime.go

```diff
@@ -340,20 +339,20 @@ func (r *runtime) InstantiateModule(
 		if start == nil {
 			continue
 		}
-		if _, err = start.Call(ctx); err != nil {
+		if _, err := start.Call(ctx); err != nil {
 			_ = mod.Close(ctx) // Don't leak the module on error.
```

Comment:
Unrelated to the bug I found, but is this correct?

It looks like this shouldn't be closed in the success case below in L347.

Reply:
I think if the module exited (even with exit code zero) it has already been closed. In that case, closing it again would not be an error.